### ISSUE TITLE
AspNetAdapter web.config

### ISSUE THREAD

**Issue by frankhale** [frankhale/AspNetAdapter#6]
Need to document in the README how to configure the web.config so that all requests are processed by the adapter.

``` xml
<system.webServer>
    <handlers>
        <add name="AspNetAdapterHandler" path="*" verb="*" type="AspNetAdapter.AspNetAdapterHandler"/>
    </handlers>
    <modules>
        <add name="AspNetAdapterModule" type="AspNetAdapter.AspNetAdapterModule"/>
    </modules>
</system.webServer>
```

**Comment by frankhale** [comment]
This information is also in the comments at the top of AspNetAdapter.cs.

**Comment by frankhale** [comment]
Closed via 9fce6dfbe4b5ee02a9da0528e5fc050959d97905
